api/envoy/api/v2/listener/udp_listener_config.proto

```diff
@@ -33,4 +34,17 @@ message UdpListenerConfig {
 }
 
 message ActiveRawUdpListenerConfig {
+  enum UdpShutDownMode {
```

Comment:
I'm not completely positive we want to bother with this kill switch for raw UDP right now. Maybe defer that for later? If we do it here, we should de-dup the messages and have a common configuration message of some type.